sim/src/image.rs

```diff
@@ -87,11 +90,11 @@ impl ImagesBuilder {
             dev_id: slot1_dev_id,
         };
 
-        ImagesBuilder {
+        Some(ImagesBuilder {
```

Comment:
Does it ever return `None` though?

Reply:
Yes, there are several earlier returns of None.

Reply:
Ok, I see you added them with another commit (a15e8f8ca2)...